src/ontology/mondo-edit.obo

```diff
@@ -228658,6 +228664,7 @@ id: MONDO:0011688
 name: muscular dystrophy-dystroglycanopathy type B5
 def: "A congenital muscular dystrophy characterized by autosomal recessive inheritance of muscular dystrophy with variable penetrance of intellectual disability and structural brain abnormalities that has material basis in homozygous or compound heterozygous mutation in the FKRP gene on chromosome 19q13.3." [DOID:0110635, PMID:11592034, PMID:14652796]
 synonym: "congenital muscular dystrophy 1C" EXACT [DOID:0110635]
+synonym: "congenital muscular dystrophy-FKRP related" EXACT CLINGEN_PREFERRED []
```

Comment:
could you please add a dbxref here:
https://clinicalgenome.org/affiliation/50061/